docs/element-definitions.md

```diff
@@ -123,10 +123,10 @@ The Resource representation with its available transitions and its data.
 #### Properties
 
 - `element`: resource (string, fixed)
-- `attributes` (object)
+- `attributes`
     - `href` (Templated Href) - URI Template of this resource.
     - `hrefVariables` (Href Variables) - Definition of URI Template variables used in the `href` property.
-- `content` (array)
+- `content` (array, fixed-type)
```

Comment:
Yeah, if we generated a JSON Schema from this structure. It wouldn't allow future values so I agree with @w-vi we shouldn't use fixed-type as that disallows future change.